src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -191,7 +191,7 @@ mkNameCompItem origName origMod thingType isInfix docs !imp = CI{..}
     insertText = case isInfix of
             Nothing -> case getArgText <$> thingType of
                             Nothing -> label
-                            Just argText -> label <> " " <> argText
+                            Just argText -> (stripPrefix label) <> " " <> argText
```

Comment:
Why not change `stripAutogenerated` so that it applies `stripPrefix` to the `insertText` field too?

https://github.com/haskell/ghcide/blob/ee184ebd7d9f76711130d1f9dc5596ac0511c297/src/Development/IDE/Plugin/Completions/Logic.hs#L641-L645

Reply:
Yes. That was the initial proposed solution that was discussed https://github.com/haskell/haskell-language-server/issues/682.   But, the `insertText` of record snippets have `:` in them and that would break record snippet completions.  

In addition, I feel doing this in a targetting manner while constructing `label`  or `insertText` is desirable and give more control during construction.

Reply:
Seems reasonable